projects/go-lib/src/lib/components/go-modal/go-modal.component.ts

```diff
@@ -117,6 +93,37 @@ export class GoModalComponent implements OnInit, OnDestroy {
     return { 'go-modal--visible': this.opened };
   }
 
+  private setModalProperties(componentRef: ComponentRef<{}>): void {
+    if (this.currentComponent.modalOptions) {
```

Comment:
For clarity, this `if` determines that if a `modalOptions` object exists on the component object that was passed in that those properties will be used to render the modal. It also means that it will ignore any properties in the `bindings` object that might pertain to the modal.

This is by design. We either use the old way or the new way, not both.